stream_file.go

```diff
@@ -65,36 +82,49 @@ func (sf *StreamFile) WriteAll(records [][]string) error {
 	return sf.zipWriter.Flush()
 }
 
-func (sf *StreamFile) write(cells []string) error {
+// documentation for the c.t (cell.Type) attribute:
```

Comment:
It is probably worth mentioning in this comment that specifying "Shared String", won't make a shared string since it is impossible during streaming.